test/CMakeLists.txt

```diff
@@ -4,11 +4,20 @@ set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 set(BUILD_MOCK ON CACHE BOOL "" FORCE)
 set(CMAKE_POLICY_DEFAULT_CMP0048 NEW)
 
-add_subdirectory(
-  "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.11.0"
-  "${CMAKE_CURRENT_BINARY_DIR}/prefix")
-
-include_directories(SYSTEM "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.11.0/googletest/include")
+find_package(GTest)
+if (NOT GTEST_FOUND)
+    add_subdirectory(
+      "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.11.0"
+      "${CMAKE_CURRENT_BINARY_DIR}/prefix")
+    include_directories(SYSTEM "${CMAKE_CURRENT_SOURCE_DIR}/gtest-1.11.0/googletest/include")
+else()
+    add_subdirectory(
+      "/usr/src/googletest"
```

Comment:
This seems very unix-specific, and also maybe requires the package to be stored in a specific location? Is there a more neutral way to use it?

Reply:
As I wrote in https://github.com/jbeder/yaml-cpp/pull/1035#discussion_r780800274, this is indeed wrong, even for linux. Just `find_package(GTest)` should be sufficient.

Reply:
ok done